tailcfg/tailcfg.go

```diff
@@ -585,6 +594,18 @@ type Hostinfo struct {
 	Userspace       opt.Bool       `json:",omitempty"` // if the client is running in userspace (netstack) mode
 	UserspaceRouter opt.Bool       `json:",omitempty"` // if the client's subnet router is running in userspace (netstack) mode
 
+	// Location represents geographical location data about a
+	// Tailscale host. Location is optional and only set if
+	// explicitly declared by a node.
+	Location *Location `json:",omitempty"`
+
+	// ExitNodePriority indicates the priority an exit node is given.
+	// A higher value indicates that the exit node is more preferable
+	// for use.
+	// A value of 0 means the exit node does not have a priority
+	// preference. A negative int is not allowed.
+	ExitNodePriority int `json:",omitempty"`
```

Comment:
I don't think HostInfo is the right place for this field. This should be on the node object. It is not declared by a node, but is instead declared by the coordination server